x/ccv/provider/keeper/keeper.go

```diff
@@ -1300,3 +1300,32 @@ func (k Keeper) DeleteConsumerCommissionRate(
 	store := ctx.KVStore(k.storeKey)
 	store.Delete(types.ConsumerCommissionRateKey(chainID, providerAddr))
 }
+
+// SetMinimumPowerInTopN sets the minimum power required for a validator to be in the top N
+// for a given consumer chain.
+func (k Keeper) SetMinimumPowerInTopN(
+	ctx sdk.Context,
+	chainID string,
```

Comment:
Why do we need the `chainID`?
Only reason that `ComputeMinPowerInTopN` has a `chainID` argument is for logging purposes.

Reply:
Because different chains have different top N, so naturally have different min power to be forced to validate